src/app/components/popups/sb-tutorial-popup/sb-tutorial-popup.component.ts

```diff
@@ -30,4 +35,21 @@ export class SbTutorialPopupComponent implements OnInit {
         this.popoverCtrl.dismiss({continueClicked});
     }
 
+    private translateMessage(messageConst: string, fields?: string | any): string {
```

Comment:
This is already there in commonutil service right , Why a separate method ?

Reply:
Because i am getting circular dependecy issue when i called common util service method,showed others too..